src/components/Grid/Grid.vue

```diff
@@ -16,8 +16,8 @@ THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
     <svg ref="grid" id="svg-grid">
       <g class="axis axis--x"></g>
       <g class="axis axis--y"></g>
-      <g class="images"></g>
-      <g class="polygons"></g>
+      <g class="images" data-transform-plz></g>
```

Comment:
What is the purpose of this creatively named attribute?

Reply:
During a zoom, we don't do a full re-render because it would fire many times and slow down the event loop. Instead, we do a visual transformation, to keep the elements in the correct position on the page (`translateEntities()` does this work).

It used to be just `.images` and `.polygons` that needed to be translated but, now that mid-draw artifacts can persist through a zoom or pan, we needed to add them as well. Rather than listing all the selectors for anyone who might want to be translated, we use `[data-transform-plz]` to let elements opt-in and say "I would like to be translated when the user pans, plz".